src/material/core/tokens/m2/mdc/_progress-spinner.scss

```diff
@@ -0,0 +1,56 @@
+@use 'sass:map';
+@use '../../../theming/theming';
+@use '../../token-utils';
+
+// The prefix used to generate the fully qualified name for tokens in this file.
+$prefix: (mdc, progress-spinner);
+
+// Tokens that can't be configured through Angular Material's current theming API,
+// but may be in a future version of the theming API.
+//
+// Tokens that are available in MDC, but not used in Angular Material should be mapped to `null`.
+// `null` indicates that we are intentionally choosing not to emit a slot or value for the token in
+// our CSS.
+@function get-unthemable-tokens() {
+  @return (
+    active-indicator-width: 4px,
+    size: 48px,
+    // =============================================================================================
+    // = TOKENS NOT USED IN ANGULAR MATERIAL                                                       =
+    // =============================================================================================
+    four-color-active-indicator-one-color: null,
```

Comment:
Add a comment explaining why they're not used (e.g. `// Angular Material does not support four color spinners`)